main.py

```diff
@@ -337,6 +342,50 @@ def isPaused(self):
 	def isRunning(self):
 		return self._p is not None # FIXME: Check subproc state
 
+class AppVeyor(object):
+	def __init__(self, repo, branch, user):
+		self.gjson = json.loads(requests.get("https://ci.appveyor.com/api/projects/" + user + "/" + repo + "/branch/" + branch).text)
```

Comment:
nit: Possibly sanitize URL, as they can not:

* Contain slashes themselves, unless this is actually supported by AppVeyor
* Other URL related special symbols which might screw us over `?` `&` `#`, ...

Not important until we allow users to change the source address, hence nitpicking.